<ui version="4.0" >
 <class>Widgets_TableGraph</class>
 <widget class="QWidget" name="Widgets_TableGraph" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QComboBox" name="graphSwitcher" />
   </item>
   <item>
    <widget class="QStackedWidget" name="graphStack" />
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
